### PR TITLE
Feature/integrate with sms service to allow sending messages from groups

### PR DESCRIPTION
**Pull request by AbelHristodor**

### CODE REVIEW

**Comment by AbelHristodor** [comment]
Forgot to put the ID on the branch, haha sorry :)

File: samaritan/templates/samaritan/views/group_members_view.html [hunk 1]
```diff
@@ -20,6 +20,11 @@ <h3>Group: <i id="group-name"></i></h3>
       <button type="button" class="btn btn-info btn-md" id="email-group-members">
         <i class="fa fa-envelope-o fa-fw"></i> Email
       </button>
+      
+      <!-- Button sms modal-->
+      <button type="button" class="btn btn-info btn-md" id="sms-group-members">
```
Comment (Silvian) [inline samaritan/templates/samaritan/views/group_members_view.html]: Can you make this button green? use btn-success I believe.

Comment (AbelHristodor) [reply samaritan/templates/samaritan/views/group_members_view.html]: Done

File: samaritan/templates/samaritan/views/group_members_view.html [hunk 2]
```diff
@@ -20,6 +20,11 @@ <h3>Group: <i id="group-name"></i></h3>
       <button type="button" class="btn btn-info btn-md" id="email-group-members">
         <i class="fa fa-envelope-o fa-fw"></i> Email
       </button>
+      
+      <!-- Button sms modal-->
+      <button type="button" class="btn btn-success btn-md" id="sms-group-members">
+        <i class="fa fa-comment fw" aria-hidden="true"></i> Message
```
Comment (Silvian) [inline samaritan/templates/samaritan/views/group_members_view.html]: sorry I missed that. It needs to be fa-fw not fw

Comment (AbelHristodor) [reply samaritan/templates/samaritan/views/group_members_view.html]: done

**Pull request merged**